diff --git a/BUILD.gn b/BUILD.gn
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1485,6 +1485,11 @@ if (angle_has_frame_capture) {
   }
 }
 
+angle_static_library("libANGLE_static") {
+  complete_static_lib = true
+  public_deps = [ ":libANGLE" ]
+}
+
 angle_static_library("libGLESv2_thin_static") {
   sources = libglesv2_entry_point_sources
   if (angle_enable_cl) {
@@ -1498,6 +1503,7 @@ angle_static_library("libGLESv2_thin_static") {
 }
 
 angle_static_library("libGLESv2_static") {
+  complete_static_lib = true
   sources = libglesv2_sources
   public_deps = [ ":libGLESv2_thin_static" ]
 }
